### PR TITLE
import added

### PR DESCRIPTION
**Pull request by ArtemKrayevskiy**

### CODE REVIEW

**Comment by roman-duck** [comment]
Manual test result:
```
FileNotFoundError: [Errno 2] No such file or directory: 'var/import/books.csv'
```

**Comment by roman-duck** [comment]
@ArtemKrayevskiy when I created 'var/import' manually, the exception has gone.
When uploading and saving to the storage is successful (or not), the result flash message on the 'index' page should be shown

File: templates/index.html [hunk 1]
```diff
@@ -121,7 +121,7 @@
         No books found
       </div>
       <p>Upload new data</p>
-      <form action="#" method="post">
+      <form action="#" method="post" enctype="multipart/form-data">
```
Comment (roman-duck) [inline templates/index.html:124]: action="#" may be removed

File: index.py [hunk 2]
```diff
@@ -31,8 +44,23 @@ def home():
...
+            flash('No file part')
+            return redirect(request.url)
+        file = request.files['file']
+        if file.filename == '':
+            flash('No selected file')
+            return redirect(request.url)
+        if file and allowed_file(file.filename):
+            filename = secure_filename(file.filename)
+            file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
+            import_service = FileImport()
```
Comment (mvchn) [inline index.py:60]: I woluld like to use import like this:
```
import_service = FileImport(app.config['UPLOAD_FOLDER'])
import_service.process_file(file, filename)
```
It allows us to remove **os** dependency from index.py

**Review by mvchn — CHANGES_REQUESTED** [review]
It is better to append (not to rewrite) data in the storage.
The ID field in the storage should be an auto-increment value.

File: file.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                            flag = True
+                    except:
+                        continue
+            if not flag:
+                result.append(book)
+            flag = False
+        with open(os.path.join("var/import/" + file_name), 'w') as file:
+            file.write(result[0])
+            for i in result[1:]:
+                file.write(str(i.id) + "," + i.title + ',' + i.author + ',' + str(i.year) + '\n')
```
Comment (roman-duck) [inline file.py:38]: no new line at the end of the file

**Comment by roman-duck** [comment]
On uploading a new file with books data a message shows:

"Your file was imported successfully. 9 unique books imported"

**Actual result:**
Imported books are not present on the index page

**Expected result:**
Imported books are present on the index page

File: file.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    def get_file_lines(self, filename):
+        with open(self.path + filename, 'r') as file_object:
+            lines = file_object.read().splitlines()
+        return lines
+
+    def save_file(self, file, file_name):
+        file.save(os.path.join(self.path, file_name))
+        with open(os.path.join("var/import/" + file_name)) as file:
+            file_containment = file.readlines()
+            print(file_containment)
```
Comment (roman-duck) [inline file.py:24]: looks like debug print

**Comment by ArtemKrayevskiy** [comment]
Can I save the imported file in var/import and copy the containment to the books.csv in var/data?

**Comment by roman-duck** [comment]
of course, you can

Here is a sample from the original issue:

`system should save the file to the /var/import directory, parse it, and save parsed data to the storage.`

**Review by roman-duck — APPROVED** [review]

**Review by mvchn — APPROVED** [review]

**Pull request merged**